src/libs/constants/social-media-icons.ts

```diff
@@ -0,0 +1,54 @@
+import FacebookIcon from "@mui/icons-material/Facebook";
+import InstagramIcon from "@mui/icons-material/Instagram";
+import MailOutlineSharpIcon from "@mui/icons-material/MailOutlineSharp";
+
+import { TwitterIcon } from "~/assets/icons/twitter-icon.tsx";
+import { colors } from "~/libs/constants/color.ts";
+
+const socialMediaDataLogIn = [
+	{
+		IconComponent: InstagramIcon,
+		backgroundColor: colors.BLACK,
+		href: "#",
+		iconColor: colors.WHITE,
+	},
+	{
+		IconComponent: FacebookIcon,
+		backgroundColor: colors.BLACK,
+		href: "#",
+		iconColor: colors.WHITE,
+	},
+	{
+		IconComponent: MailOutlineSharpIcon,
+		backgroundColor: colors.BLACK,
+		href: "#",
+		iconColor: colors.WHITE,
+	},
+];
+const socialMediaDataLogOut = [
```

Comment:
empty line above

Reply:
It is a linter problem ,we will solve it later